File: codes/clase_01/suma.c
#include <stdio.h>

float main(void) {
	float a;
	float b;
	float c;

	printf("Ingrese el valor a: ");  
	scanf("%f", &a);
	printf("Ingrese el valor b: ");  
	scanf("%f", &b);
	c = a+b;
	printf("La suma es: %f\n", c);
	return 0;
}

File: ejemplos/python/suma.f90
program suma

implicit none
real :: a
real :: b
real :: c

write(*,*) 'Ingrese el valor a: '
read(*,*) a
write(*,*) 'Ingrese el valor b: '
read(*,*) b

c = a + b

write(*,*) 'La suma es: ', c

end program suma
